doc/releases/changelog-dev.md

```diff
@@ -174,6 +174,28 @@
   }
   ```
 
+  * A new :func:`~catalyst.passes.decompose_arbitrary_ppr` pass has been added to the `catalyst.passes` module.
+    This will decompose into a collection of PPRs, PPMs and a single-qubit arbitrary PPR 
```

Comment:
This will decompose an arbitrary angle PPR into a collection of PPRs, PPMs and a single-qubit arbitrary PPR in the Z basis.